hass-configurator/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+CONFIG_PATH=/data/options.json
+
+python3 /configurator.py ${CONFIG_PATH}
```

Comment:
You need not have a `run.sh` or modify the `configurator.py` to +a. Add this into Dockerfile and remove this `run.sh` file:
```
CMD ["python3", "/configurator.py", "/data/options.json"]
```